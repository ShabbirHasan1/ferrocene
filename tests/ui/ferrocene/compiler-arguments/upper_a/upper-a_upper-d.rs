// When both -A and -D are specified for the same lint, the last one overrides
// the previous. So here, -D overrides -A, denying the lint.
//
// check-fail
// compile-flags: -A overflowing_literals -D overflowing_literals

fn main() {
    let x: u8 = 1000; //~ ERROR
}

// ferrocene-annotations: um_rustc_A
