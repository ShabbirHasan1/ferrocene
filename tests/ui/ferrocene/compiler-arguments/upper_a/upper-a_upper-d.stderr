error: literal out of range for `u8`
  --> $DIR/upper-a_upper-d.rs:8:17
   |
LL |     let x: u8 = 1000;
   |                 ^^^^
   |
   = note: the literal `1000` does not fit into the type `u8` whose range is `0..=255`
   = note: requested on the command line with `-D overflowing-literals`

error: aborting due to previous error

